pom.xml

```diff
@@ -6,7 +6,7 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.github.ericsson</groupId>
     <artifactId>eiffel-intelligence</artifactId>
-    <version>3.0.1-SNAPSHOT</version>
+    <version>3.0.2-SNAPSHOT</version>
```

Comment:
Have we released 3.0.1? What version number do we plan to use in next release?

Reply:
3.0.1 was not releases, my mistake will revert the pom.xml